ui/event_handlers.go

```diff
@@ -92,42 +95,86 @@ func (r *Handler) UpdateVulnerabilityDatabaseHandler(ctx context.Context, fr *fr
+func scanningAndSummaryLines(fr *frame.Frame) (scanningLine, summaryLine *frame.Line, err error) {
+	scanningLine, err = fr.Append()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	summaryLine, err = fr.Append()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return scanningLine, summaryLine, nil
+}
+
+func assembleProgressMonitors(m *matcher.Monitor) []progress.Monitorable {
+	return []progress.Monitorable{
+		m.PackagesProcessed,
+		m.VulnerabilitiesDiscovered,
+		m.VulnerabilitiesCategories.Unknown,
+		m.VulnerabilitiesCategories.Low,
+		m.VulnerabilitiesCategories.Medium,
+		m.VulnerabilitiesCategories.High,
+		m.VulnerabilitiesCategories.Critical,
+		m.VulnerabilitiesCategories.Fixed,
+	}
+}
+
 func (r *Handler) VulnerabilityScanningStartedHandler(ctx context.Context, fr *frame.Frame, event partybus.Event, wg *sync.WaitGroup) error {
 	monitor, err := grypeEventParsers.ParseVulnerabilityScanningStarted(event)
 	if err != nil {
 		return fmt.Errorf("bad %s event: %w", event.Type, err)
 	}
 
-	line, err := fr.Append()
+	scanningLine, summaryLine, err := scanningAndSummaryLines(fr)
 	if err != nil {
 		return err
 	}
 
 	wg.Add(1)
 
+	monitors := assembleProgressMonitors(monitor)
+
 	_, spinner := startProcess()
-	stream := progress.StreamMonitors(ctx, []progress.Monitorable{monitor.PackagesProcessed, monitor.VulnerabilitiesDiscovered}, 50*time.Millisecond)
+	stream := progress.StreamMonitors(ctx, monitors, 50*time.Millisecond)
+
 	title := tileFormat.Sprint("Scanning image...")
+	title2 := tileFormat.Sprint("Summary")
 
-	formatFn := func(val int64) {
+	formatFn := func(total, unknown, low, medium, high, critical, fixed int64) {
 		spin := color.Magenta.Sprint(spinner.Next())
-		auxInfo := auxInfoFormat.Sprintf("[vulnerabilities %d]", val)
-		_, _ = io.WriteString(line, fmt.Sprintf(statusTitleTemplate+"%s", spin, title, auxInfo))
+		auxInfo := auxInfoFormat.Sprintf("[vulnerabilities %d]", total)
+		_, _ = io.WriteString(scanningLine, fmt.Sprintf(statusTitleTemplate+"%s", spin, title, auxInfo))
+
+		auxInfo2 := auxInfoFormat.Sprintf("[Critical: %d, High: %d, Medium: %d, Low: %d, Unknown: %d, Fixed: %d]", critical, high, medium, low, unknown, fixed)
+		_, _ = io.WriteString(summaryLine, fmt.Sprintf(statusTitleTemplate+"%s", spin, title2, auxInfo2))
 	}
 
 	go func() {
 		defer wg.Done()
 
-		formatFn(0)
+		formatFn(0, 0, 0, 0, 0, 0, 0)
 		for p := range stream {
-			formatFn(p[1])
+			formatFn(p[1], p[2], p[3], p[4], p[5], p[6], p[7])
 		}
 
 		spin := color.Green.Sprint(completedStatus)
 		title = tileFormat.Sprint("Scanned image")
 		auxInfo := auxInfoFormat.Sprintf("[%d vulnerabilities]", monitor.VulnerabilitiesDiscovered.Current())
-		_, _ = io.WriteString(line, fmt.Sprintf(statusTitleTemplate+"%s", spin, title, auxInfo))
+		_, _ = io.WriteString(scanningLine, fmt.Sprintf(statusTitleTemplate+"%s", spin, title, auxInfo))
+
+		auxInfo2 := auxInfoFormat.Sprintf(
+			"[Critical: %d, High: %d, Medium: %d, Low: %d, Unknown: %d, Fixed: %d]",
```

Comment:
A quick UX note: maybe we don't show the types that have 0 reported vulnerabilities so this line may be shorter?

Reply:
In my personal thought: Always show all columns.  Show all, even if they have "0 count". 
Breaking symmetry, can be troublesome, in my opinion.

Reply:
I didn't have a strong opinion about this, just a suggestion and fine as-is 👍